modules/nodes-pods-autoscaling-custom-install.adoc

```diff
@@ -0,0 +1,116 @@
+
+. On the *Install Operator* page, ensure that the *All namespaces on the cluster (default)* option
+is selected for *Installation Mode*. This installs the Operator in all namespaces.
+
+. Ensure that the *openshift-keda* namespace is selected for *Installed Namespace*.  
+
+. Click *Install*.
+
+. Verify the installation by listing the Custom Metrics Autoscaler Operator components:
+
+.. Navigate to *Workloads* -> *Pods*.
+
+.. Select the `openshift-keda` project from the drop-down menu and verify that the `custom-metrics-autoscaler-operator-*` pod is running.
+
+.. Navigate to *Workloads* -> *Deployments* to verify that the `custom-metrics-autoscaler-operator` deployment is running.
+
+. Optional. Verify the installation in the {product-title} CLI using the following commands:
++
+[source,terminal]
+----
+$ oc get all -n openshift-keda
+----
++
+The output appears similar to the following:
++
+.Example output
+[source,terminal]
+----
+NAME                                                      READY   STATUS    RESTARTS   AGE
+pod/custom-metrics-autoscaler-operator-5fd8d9ffd8-xt4xp   1/1     Running   0          18m
+
+NAME                                                 READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/custom-metrics-autoscaler-operator   1/1     1            1           18m
+
+NAME                                                            DESIRED   CURRENT   READY   AGE
+replicaset.apps/custom-metrics-autoscaler-operator-5fd8d9ffd8   1         1         1       18m
+----
+
+. Install the `KedaController` custom resource, which creates the required CRDs:
+
+.. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+.. Click *Custom Metrics Autoscaler*.
+
+.. On the *Operator Details* page, click the *KedaController* tab.
+
+.. On the *KedaController* tab, click *Create KedaController* and edit the file.
++
+[source,yaml]
+----
+kind: KedaController
+apiVersion: keda.sh/v1alpha1
+metadata:
+  name: keda
+  namespace: openshift-keda
+spec:
+  watchNamespace: '' <1>
+  operator:
+    logLevel: info <2>
+    logEncoder: console <3>
+  metricsServer:
+    logLevel: '0' <4>
+  serviceAccount: {}
+----
+<1> Specifies the namespaces that the custom autoscaler should watch. Enter names in a comma-separated list. Omit or set empty to watch all namespaces. The default is empty.
+<2> Specifies the level of verbosity for the Custom Metrics Autoscaler Operator log messages. The allowed values are `debug`, `info`, `error`. The default is `info`.
+<3> Specifies the logging format for the Custom Metrics Autoscaler Operator log messages. The allowed values are `console` or `json`. The default is `console`.
+<4> Specifies the logging level for the Custom Metrics Autoscaler Metrics Server. The allowed values are `0` for `info` and `4` or `debug`. The default is `0`.
++
+See the link:https://github.com/kedacore/keda-olm-operator[KEDA OLM Operator] document for all available parameters.
```

Comment:
Just checking if you cleared this GH link with Eric Rich?

Reply:
@bergerhoffer 
Yes! That's what I wanted you to ask!